website/conferences/views.py

```diff
@@ -86,6 +87,7 @@ def add_poster_by_email(conference, message):
                 verification_key=user.verification_key,
                 _absolute=True,
             )
+            user.date_last_login = datetime.utcnow()
```

Comment:
It might make sense to put this in `User.create_confirmed`, but I'm not sure. I think this is fine here for now.